src/corona_hakab_model/util.py

```diff
@@ -32,6 +32,17 @@ def upper_bound(d):
     return d.b + d.kwds.get("loc", 0)
 
 
+def get_numpy_uniform_dist(l_bound=None, u_bound=None):
+    assert l_bound is not None and u_bound is None\
```

Comment:
why asserting u_bound is None?

i see that the variables names are confusing because sometimes the first variable is lower bound, and some times it is the higher bound. 
why not just give them generic names ...(a, b=None ).
this way you enforce at least one value without the assert, and the names don't get confused

Reply:
I somewhat agree ( I do not like a and b cause they are not such meaningfull name, instead I added a doc)